joomla/joomla-cms
# JLayout to render debug backtrace; Clickable links for XDebug

This Pull Request is a redo of #3780, see that PR for more information
## Modifications

This PR adds a JLayout file for the callstack backtrace. The debug plugin and isis template include this layout now.
On request of @roland-d in the original PR I also add clickable links for XDebug session. If the flag `xdebug.file_link_format` is set in the php.ini the location link will be clickable.
## Test instructions
1. Go to the Global Configuration, system tab and enable "Debug system"
2. Type `/administrator/index.php?option=foo` in the browser and you will see the following error page
   ![isis-error](https://cloud.githubusercontent.com/assets/2659866/15097382/9e68cc8e-1519-11e6-8144-8f5b0c4f3d5b.png)
3. If you have XDebug you will see the following error page
   ![isis-error-xdebug](https://cloud.githubusercontent.com/assets/2659866/15097393/f579a1e2-1519-11e6-9ef2-bf17ef812737.png)
